boot/zephyr/main.c

```diff
@@ -159,6 +159,8 @@ struct arm_vector_table {
     uint32_t reset;
 };
 
+extern void sys_clock_disable(void);
```

Comment:
I think we should `#include <drivers/timer/system_timer.h>` rather than having this extern ourselves. This way, the function will correctly be stubbed out on configurations that do not have this function.

Reply:
Unfortunately, `sys_clock_disable` is not part of the definition (at least not in Zephyr 2.7,0 and 2.7.1).
Is this version intended to run on Zephyr >= 3.0.0? If so, I'll drop the pull request.

Reply:
@kvaellning Mcuboot is expected to run with zephyr more less of the same time.
Exact version is part of the zephyr manifest.